aemedge/scripts/utils.js

```diff
@@ -24,4 +25,42 @@ function containerize(container, targetClass) {
   }
 }
 
-export { formatDate, containerize };
+async function fetchTagList(prefix = 'default') {
+  window.tags = window.tags || {};
```

Comment:
Instead of making global object can we save it in local storage so we don't fetch it on every page?

Reply:
... used the same pattern like done for placeholders, but yes. I created a follow up #502